README.rst

```diff
@@ -1,5 +1,6 @@
 django-powerdns-dnssec
 ===================================
+# XXX from where that `-dnssec` suffix comes from..?
```

Comment:
it all started with django-powerdns (https://bitbucket.org/peternixon/django-powerdns/wiki/Home). Then Łukasz Langa forked it under django-powerdns-dnssec (as denoted on pypi: "This is an updated and enhanced fork of django-powerdns which looks abandoned as of November 2012."). So django-powerdns-dnssec provides DNSSEC tables support. 

Now we're creating DNSaaS based on powerdns rather than powerdns manager in admin - it has permissions module, jira integration, API etc.